ui/components-react/src/test/filter-builder/FilterBuilderState.test.ts

```diff
@@ -726,6 +880,212 @@ describe("usePropertyFilterBuilder", () => {
     });
   });
 
+  describe("range rule", () => {
+    it("converts `>=` and `<=` rules into `Between` rule", () => {
+      const { result } = renderHook(() =>
+        usePropertyFilterBuilder({
+          initialFilter: {
+            operator: PropertyFilterRuleGroupOperator.And,
```

Comment:
maybe `PropertyFilterBuilderRuleGroup.operator` should also be changed to accept strings like `PropertyFilterBuilderRule.operator`?